### PR TITLE
NMS-15853: More BMP docs related to distributed processing.

### PR DESCRIPTION
**Pull request by j-white**
JIRA: https://opennms.atlassian.net/browse/NMS-15853

### CODE REVIEW

**Comment by j-white** [comment]
@dino2gnt with this and the examples in the BMP playground: https://github.com/opennms-forge/bmp-playground/pull/6, is there anything that needs more clarity?

File: docs/modules/reference/pages/telemetryd/protocols/bmp/bmp.adoc [hunk 1]
```diff
@@ -7,4 +7,12 @@ The integration in {page-component-title} lets you use these route views, status
...
 image::bmp/bmp_integration_overview.png[width=70%]
 
-The following sections provide details about associated parsers, adapters, and listeners.
\ No newline at end of file
+The following sections provide details about associated parsers, adapters, and listeners.
+
+== Distributed BMP processing with Sentinel
+
+Sentinel can be used to process BMP statistics ingested by one or more Minions in order to alleviate load on the core.
+All of the BMP related adapters can be ran on Sentinel, allowing you to generate events, metrics and ingest route changes.
```
Comment (Bonrob2) [inline docs/modules/reference/pages/telemetryd/protocols/bmp/bmp.adoc:15]:
```suggestion
All of the BMP-related adapters can run on Sentinel, allowing you to generate events and metrics, and ingest route changes.
```

File: docs/modules/reference/pages/telemetryd/protocols/bmp/bmp.adoc [hunk 2]
```diff
@@ -7,4 +7,12 @@ The integration in {page-component-title} lets you use these route views, status
 .BMP integration overview
 image::bmp/bmp_integration_overview.png[width=70%]
 
-The following sections provide details about associated parsers, adapters, and listeners.
\ No newline at end of file
+The following sections provide details about associated parsers, adapters, and listeners.
+
+== Distributed BMP processing with Sentinel
+
+Sentinel can be used to process BMP statistics ingested by one or more Minions in order to alleviate load on the core.
```
Comment (Bonrob2) [inline docs/modules/reference/pages/telemetryd/protocols/bmp/bmp.adoc:14]:
```suggestion
You can use Sentinel to process BMP statistics ingested by one or more Minions in order to alleviate load on the core.
```

File: docs/modules/reference/pages/telemetryd/protocols/bmp/bmp.adoc [hunk 3]
```diff
@@ -7,4 +7,12 @@ The integration in {page-component-title} lets you use these route views, status
...
+All of the BMP related adapters can be ran on Sentinel, allowing you to generate events, metrics and ingest route changes.
+In this mode, events are sent back to the message broker to be processed by the core and metrics are persisted directly to the time-series database.
```
Comment (Bonrob2) [inline docs/modules/reference/pages/telemetryd/protocols/bmp/bmp.adoc:16]:
```suggestion
In this mode, events are sent back to the message broker for the core to process and metrics are persisted directly to the time series database.
```

File: docs/modules/reference/pages/telemetryd/protocols/bmp/bmp.adoc [hunk 4]
```diff
@@ -7,4 +7,12 @@ The integration in {page-component-title} lets you use these route views, status
...
+
+TIP: The https://github.com/OpenNMS-forge/bmp-playground[BMP Playground] provides a virtual lab that can be used to simulate BGP, BMP, and see how the OpenNMS components can be configured at runtime.
```
Comment (Bonrob2) [inline docs/modules/reference/pages/telemetryd/protocols/bmp/bmp.adoc:18]:
```suggestion
TIP: The https://github.com/OpenNMS-forge/bmp-playground[BMP Playground] provides a virtual lab that you can use to simulate BGP, BMP, and to see how the {page-component-title} components can be configured at runtime.
```

**Review by Bonrob2 — CHANGES_REQUESTED** [review]
a few minor changes. Thanks.

**Review by Bonrob2 — APPROVED** [review]
docs LGTM! Thanks!

**Comment by dino2gnt** [comment]
LGTM.  Should this go to a `foundation` ?

**Review by Bonrob2 — APPROVED** [review]
LGTM!

**Pull request merged**